src/webgl/gl-constants.js

```diff
@@ -0,0 +1,870 @@
+// WEBGL BUILT-IN TYPES
```

Comment:
These are some incredible work! Did you actually go through the GL header and copied them line by line? wow...